packages/server-wallet/src/__test-with-peers__/crash-tolerance/crash-tolerance.test.ts

```diff
@@ -117,6 +97,9 @@ it('Create a directly-funded channel between two wallets, of which one crashes m
     turnNum: 0,
   });
 
+  await messageService.send(getMessages(resultA2));
+  await messageService.send(getMessages(resultB2));
```

Comment:
Clarifying intent: the various `messageService.send` calls are being inserted into existing tests in order to ensure that sending / receiving messages doesn't impact previous behavior?

Reply:
Yup.  The intent here was to create a new messaging service and use the `with-peers` tests to ensure that using the new messaging service didn't break anything.